src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1108,6 +1108,27 @@ describe('ReactDOMComponent', () => {
       expectDev(console.log.calls.argsFor(0)[0]).toContain('onError called');
       expectDev(console.log.calls.argsFor(1)[0]).toContain('onLoad called');
     });
+
+    it('should support onLoad and onError events on SVG element', () => {
+      var container = document.createElement('div');
+
+      spyOn(console, 'log');
+
+      ReactDOM.render(
+        <svg
+            onError={() => console.log('onError called')}
+            onLoad={() => console.log('onLoad called')}
```

Comment:
Nit pick: Should there be 2 space indentation here? It may not be necessary after the comment below.